src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/DebugProfileEnumValuesGeneratorTests.cs

```diff
@@ -32,8 +32,13 @@ public class DebugProfileEnumValuesGenerator_Tests
 
             Mock<ILaunchSettingsProvider> moqProfileProvider = new Mock<ILaunchSettingsProvider>();
             moqProfileProvider.Setup(p => p.CurrentSnapshot).Returns(testProfiles.Object);
+            var moqThreadingService = new Mock<IProjectThreadingService>();
+            moqThreadingService
+                .Setup(p => p.JoinableTaskFactory)
+                .Returns(new Threading.JoinableTaskFactory(new Threading.JoinableTaskContext()));
```

Comment:
Tip: you can use `new JoinableTaskContext().Factory` instead of new'ing up your own JTF.